api/specs.proto

```diff
@@ -101,6 +102,7 @@ message TaskSpec {
 	oneof runtime {
 		NetworkAttachmentSpec attachment = 8;
 		ContainerSpec container = 1;
+		CustomSpec custom = 10;
```

Comment:
Yes, I don't like `CustomSpec` / `custom` either. I was just pointing out why it's a wrapper object.

We thought about `RuntimeSpec`, the problem however is that we already have a `oneof runtime` in TaskSpec, e.g.

```go
 	oneof runtime {
		RuntimeSpec runtime = 10;
```

It's a bit weird to have a runtime inside a runtime.

Also, this won't compile I believe - `Spec.GetRuntime()` would be defined twice

Suggestions?